src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -19,8 +19,56 @@
          "groupId" : "com.google.cloud",
          "artifactId" : "google-cloud-bigquery",
          "version" : "0.22.0-beta"
+       }
       }
-    }
-   ]
+    ]
+  },
+  {
+    "name": "Google Cloud Datastore",
+    "documentation": "https://cloud.google.com/datastore/docs/reference/libraries",
+    "description":"Google Cloud Datastore is a NoSQL key-value database built for automatic scaling.",
+    "transport": "http",
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/datastore.svg",
+    "clients": [
+      {
+      "name":"Cloud Datastore Client Library for Java",
+      "language":"java",
+      "site":"https://cloud.google.com/datastore/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/datastore/package-summary.html",
+      "infotip":"The com.google.cloud.datastore packages",
+      "status" : "GA",  
```

Comment:
I think `releaseLevel` would be a more accurate description of the Alpha/Beta/GA values

Reply:
wikipedia seems to call this a "stage" though I can't say I've heard that before: https://en.wikipedia.org/wiki/Software_release_life_cycle#Stages_of_development

Reply:
It looks like "launch stage" is actually the terminology used by Google Cloud Platform: https://cloud.google.com/terms/launch-stages

However, we have to keep in mind that the release level of client libraries will lag behind the service.